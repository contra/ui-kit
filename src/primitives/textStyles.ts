/* eslint-disable unicorn/no-abusive-eslint-disable */
/* eslint-disable eslint-comments/no-unlimited-disable */
/* eslint-disable */

/* Generated file. Do not update manually! */
export const textStyles = {
  body: {
    regular: {
      fontSize: '16px',
      fontWeight: '400',
      lineHeight: '1.5',
    },
    small: {
      fontSize: '14px',
      fontWeight: '400',
      lineHeight: '1.429',
      letterSpacing: '0.15px',
    },
  },
  button: {
    regular: {
      fontSize: '16px',
      fontWeight: '600',
      lineHeight: '1.5',
    },
  },
  caption: {
    bold: {
      fontSize: '12px',
      fontWeight: '600',
      lineHeight: '1.333',
      letterSpacing: '0.15px',
    },
    regular: {
      fontSize: '12px',
      fontWeight: '400',
      lineHeight: '1.333',
      letterSpacing: '0.15px',
    },
  },
  codeBlock: {
    regular: {
      fontSize: '16px',
      fontWeight: '600',
      lineHeight: '1.5',
      fontFamily: 'var(--font-mono)',
    },
  },
  heading: {
    h1: {
      fontSize: '93px',
      fontWeight: '700',
      lineHeight: '1.29',
      letterSpacing: '-2.5px',
    },
    h2: {
      fontSize: '58px',
      fontWeight: '700',
      lineHeight: '1.241',
      letterSpacing: '-1px',
    },
    h3: {
      fontSize: '48px',
      fontWeight: '700',
      lineHeight: '1.25',
      letterSpacing: '-0.75px',
    },
    h4: {
      fontSize: '34px',
      fontWeight: '700',
      lineHeight: '1.294',
      letterSpacing: '-0.5px',
    },
    h5: {
      fontSize: '23px',
      fontWeight: '600',
      lineHeight: '1.391',
      letterSpacing: '-0.25px',
    },
    h6: {
      fontSize: '19px',
      fontWeight: '600',
      lineHeight: '1.474',
    },
  },
  overline: {
    regular: {
      fontSize: '10px',
      fontWeight: '500',
      lineHeight: '1.6',
      letterSpacing: '1.2px',
      textTransform: 'uppercase',
    },
  },
  subtitle: {
    regular: {
      fontSize: '16px',
      fontWeight: '600',
      lineHeight: '1.5',
      letterSpacing: '-0.15px',
    },
    small: {
      fontSize: '14px',
      fontWeight: '600',
      lineHeight: '1.429',
    },
  },
};
