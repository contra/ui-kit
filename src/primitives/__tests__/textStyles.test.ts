/**
 * This snapshot test is here to make sure we are intentional
 * about any updates to this file.
 *
 * 1. Did the format change? If so did we fix how that effects things down stream?
 * 2. Text style changes or removal need treated like a BREAKING change.
 */

import { textStyles } from '../textStyles';

describe('textStyles', () => {
  it('should match snapshot', () => {
    expect(textStyles).toMatchSnapshot();
  });
});
