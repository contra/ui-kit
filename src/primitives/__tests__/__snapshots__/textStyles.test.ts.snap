// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textStyles should match snapshot 1`] = `
Object {
  "body": Object {
    "regular": Object {
      "fontSize": "16px",
      "fontWeight": "400",
      "lineHeight": "1.5",
    },
    "small": Object {
      "fontSize": "14px",
      "fontWeight": "400",
      "letterSpacing": "0.15px",
      "lineHeight": "1.429",
    },
  },
  "button": Object {
    "regular": Object {
      "fontSize": "16px",
      "fontWeight": "600",
      "lineHeight": "1.5",
    },
  },
  "caption": Object {
    "bold": Object {
      "fontSize": "12px",
      "fontWeight": "600",
      "letterSpacing": "0.15px",
      "lineHeight": "1.333",
    },
    "regular": Object {
      "fontSize": "12px",
      "fontWeight": "400",
      "letterSpacing": "0.15px",
      "lineHeight": "1.333",
    },
  },
  "codeBlock": Object {
    "regular": Object {
      "fontFamily": "var(--font-family-mono)",
      "fontSize": "16px",
      "fontWeight": "400",
      "lineHeight": "1.5",
    },
  },
  "heading": Object {
    "h1": Object {
      "fontSize": "93px",
      "fontWeight": "700",
      "letterSpacing": "-2.5px",
      "lineHeight": "1.29",
    },
    "h2": Object {
      "fontSize": "58px",
      "fontWeight": "700",
      "letterSpacing": "-1px",
      "lineHeight": "1.241",
    },
    "h3": Object {
      "fontSize": "48px",
      "fontWeight": "700",
      "letterSpacing": "-0.75px",
      "lineHeight": "1.25",
    },
    "h4": Object {
      "fontSize": "34px",
      "fontWeight": "700",
      "letterSpacing": "-0.5px",
      "lineHeight": "1.294",
    },
    "h5": Object {
      "fontSize": "23px",
      "fontWeight": "600",
      "letterSpacing": "-0.25px",
      "lineHeight": "1.391",
    },
    "h6": Object {
      "fontSize": "19px",
      "fontWeight": "600",
      "lineHeight": "1.474",
    },
  },
  "overline": Object {
    "regular": Object {
      "fontSize": "10px",
      "fontWeight": "500",
      "letterSpacing": "1.2px",
      "lineHeight": "1.6",
      "textTransform": "uppercase",
    },
  },
  "subtitle": Object {
    "regular": Object {
      "fontSize": "16px",
      "fontWeight": "600",
      "letterSpacing": "-0.15px",
      "lineHeight": "1.5",
    },
    "small": Object {
      "fontSize": "14px",
      "fontWeight": "600",
      "lineHeight": "1.429",
    },
  },
}
`;
